views/about.erb

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Jenna Nichols: About</title>
+  <link href="data:image/x-icon;base64,AAABAAEAEBAAAAEACABoBQAAFgAAACgAAAAQAAAAIAAAAAEACAAAAAAAAAEAAAAAAAAAAAAAAAEAAAAAAAD6+vYA19KqAIl6BwDRy50AqqNyAL20cwDKxJAA8O7fAGZbBQBiVwAAgHEAALezhwB4biIAgnQAAIR0AACPgRUA+fnyAMC4eQCGdwAAh3cAAPP08ABmWwYA/v79AIp8CwBkaD0AdGkQAN3ZuQChlTgA+/z7ADk+CACUlnkAhXw2AIZ7IADm4soAhnkNAM7ImACLfQoAnJVbAIR1AADMyKEA6ebQAMjBjADr6NMAwrp/APj38wCkmUAApZlAALivbQCQghYA0cyfAH10LgBCRxIAlIchAP///gDv7uQAgHICAIaFUQD29/QAk5d5ANjTrQB+dSwA5uPKAIx+EgD5+fcAz8qmAG1xSQD///8Ag3MAAMvNvwDr6NUA7OjVAFZbLQD5+PIAhXYAAM3HlgDPyZkA5+XOAMjJuwDKxI8Ag3owAPPx5gAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQkIHSEJCQkJCQkJCQkI1QkJCRiRCQkI8Fk8lQkJCBTFCQkIvIUIyLEJCCx9CQj0tQkJCAT42CUJCQgQVABpDLkJCQkUmIBlCQkIMCClJJhBCQkJLEhJJDhIPNwpJEgJCQkJCFxISEhISEhISEhIwQkJCUCYSEhISEhISEhISNEJCQhsSEhJJSUkSEhISEydCQismEg0RKkw7A0pOTjhHQkAmEkMoQkJCQkJCQkIcHTlCBiIjQkJCQkJCQkJCQhhEQkIUQkJCQkJCQkJCQkJBTUJCQkJCQkJCQkJCQkJCMz9CQkJCQkJCQkJCQkJCHjpCQkJCQkJCQkJCQkJCQkJCQgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=" rel="icon" type="image/x-icon" />
```

Comment:
Hmmm... that's lame. I know Jennie was trying the same thing but couldn't get it to work as far as I know